src/Stryker.Core/Stryker.Core/Reporters/Json/TestFiles/JsonTestFile.cs

```diff
@@ -1,14 +1,15 @@
 using System.Collections.Generic;
+using Stryker.Core.ProjectComponents.TestProjects;
 
 namespace Stryker.Core.Reporters.Json.TestFiles
 {
-    public class TestFile
+    public class JsonTestFile
```

Comment:
I'd say the name is confusing now. I'd imagine something like "MyTests.json" instead of "MyTests.cs" :)

Reply:
This is the Class corresponding to the json schema for test reports. To avoid confusion with the in-memory model for tests in stryker I named it this. I also dislike the name. What do you suggest?

Reply:
Hmm. Maybe something like JsonTestSchema? Dunno, but the word schema could probably help ;)